packages/database/src/models/EncounterAssignedSurvey.ts

```
@@ -0,0 +1,73 @@
+        ...options,
+        tableName: 'encounter_assigned_surveys',
+        timestamps: true,
+        paranoid: true,
+        underscored: true,
+        syncDirection: SYNC_DIRECTIONS.BIDIRECTIONAL,
+      },
+    );
+  }
+
+  static initRelations(models: Models) {
+    this.belongsTo(models.Encounter, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)